README.md

```diff
@@ -1,6 +1,31 @@
 # amen
 A toolbox for algorithmic remixing, after Echo Nest Remix.
 
+# Platforms
+Amen is developed on Ubuntu 14.04 and higher.  OS X is probably workable.  Windows users should install Ubuntu.
+
+# Installation
+Amen is pretty simple, but it stands on top of some complex stuff.
+
+First, you'll need our favorite resampler: `apt-get libsamplerate`.
+
+Then, you'll need to get our own version of it `pip install git+https://github.com/bmcfee/samplerate.git`
+
+Now, if you're a serious Python nerd, you can just get it from pip:  `pip install amen`.
+
+If not, read on for faster, more user-friendly installation process.
+
+First, you should install Anaconda, (https://www.continuum.io/downloads) which will get you all of the dependencies.
+
+Then, install via pip:  `pip install amen`.
+
+# Testing the Installation
+After installation is finished, run `python amen/examples/reverse.py amen/examples/audio/amen./wav`.
+Play the resulting `reversed.wav` file, and you're on your way!
+
```

Comment:
If you pip install amen,

- Are the example scripts bundled in setup.py?  And the data?
- The user will have to go hunt through their python library path to get the example script